<launch>
  <param name="gui/color1" value="red" type="str"/>
  <param name="gui/color2" value="blue" type="str"/>
  <param name="gui/color3" value="green" type="str"/>

  <param name="gui/shape1" value="CRU" type="str"/>
  <param name="gui/shape2" value="CIR" type="str"/>

  <param name="gui/break_number" value="5" type="int"/>
 
  <node pkg="robotx_gui" type="result_gui.py" name="result_gui_display">
  </node>

</launch>
